build/math-cert-tests/math-code-test-a.py

```diff
@@ -225,7 +223,7 @@ def step21(code):
         print("Go on to the next step")
 
 
-def step22():
+def step22(n, exponent):
```

Comment:
When I run this on Colab with these two arguments, I get an error saying that `numerator` is not defined.

When I created a new test file with the suggestion below, imported it, and updated the function call in the notebook, I no longer got that name error, or errors for `denominator`, `percent`, or `exponent`.

```suggestion
def step22(n, numerator, denominator, percent, exponent):
```

Reply:
Oh, of course, not sure how this happened...